src/components/React/Controls.js

```diff
@@ -131,113 +103,68 @@ export default connect(mapStateToProps, {
 
     render() {
       return (
-        <div id="controls">
-          <div>
-            <br />
-            <div
-              style={this.props.volumeData !== "" ? {} : { display: "none" }}
-            >
-              <label>Channel</label>
-              <br />
-              <Dropdown
-                disabled={this.state.activateColorMapping}
-                value={this.state.dataCurrentChannel}
-                options={channelOptions}
-                onChange={this.channelSelectChanged}
-                placeholder="Select Channel"
+        <Container fluid id="controls">
+          <Row className="my-3">
+            <ColorMapControl width="250" />
+          </Row>
+          <Row className="my-3">
+            <OpacityControl width="250" />
+          </Row>
+
+          <Form className="mt-5">
+            <Form.Group as={Row}>
+              <Form.Label>X Slider</Form.Label>
+              <Range
+                // disabled={
+                // this.props.volumeData === "" ||
+                // this.props.volumeData === undefined
+                // ? true
+                // : false
+                // }
+                allowCross={false}
+                step={0.0009}
+                defaultValue={[0, 1]}
```

Comment:
what does the array as a default value signify here? also, why the oddly specific step size?